db/schema.rb

```diff
@@ -79,6 +72,28 @@
   add_index "pillars", ["minion_id"], name: "index_pillars_on_minion_id", using: :btree
   add_index "pillars", ["pillar"], name: "index_pillars_on_pillar", using: :btree
 
+  create_table "registries", force: :cascade do |t|
+    t.string   "name",                   limit: 255
+    t.string   "url",                    limit: 255
+    t.integer  "certificate_service_id", limit: 4
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
+  add_index "registries", ["certificate_service_id"], name: "index_registries_on_certificate_service_id", using: :btree
+
+  create_table "registry_mirrors", force: :cascade do |t|
+    t.string   "name",                   limit: 255
+    t.string   "url",                    limit: 255
+    t.integer  "registry_id",            limit: 4
+    t.integer  "certificate_service_id", limit: 4
```

Comment:
This should be removed, the proxy model will take this into account by linking `certificate_id` with a `service_type` and a `service_id` (in this case `RegistryMirror` as type and `registry_mirrors`.`id` as id).